app/main/appium-method-handler.js

```diff
@@ -101,7 +104,11 @@ export default class AppiumMethodHandler {
     // Give the source/screenshot time to change before taking the screenshot
     await Bluebird.delay(500);
 
-    let sourceAndScreenshot = await this._getSourceAndScreenshot();
+    let sourceAndScreenshot;
```

Comment:
can we avoid code duplication here?

Reply:
Yeah I'll make it so there's one main method.